paper.tex

```diff
@@ -476,137 +476,162 @@ \section*{Appendix}
 distant past. By having time units increase into the past, we avoid the need to
-choose an arbitrary point in history as ``time zero'', so that instead zero
-refers to ``now'' or the final generation of a simulation. A natural
+choose an arbitrary point in history as ``time zero''. A natural
 specification for time units is in generations, although other time units are
 permitted, such as years, accompanied by the generation time.
 
 Population sizes are given as numbers of individuals, and details
 such as ploidy levels are considered external to the model.
-We therefore focus on the number of individuals as opposed to the number of genomes.
+We therefore focus on the number of individuals as opposed
+to the number of genome copies.
 Sizes and mating system details are specified for each population within
 epochs.
 Epochs are contiguous time intervals that define
 the existence interval of the population. Each epoch specifies the population size
 over that interval, which can be a constant value or function defined by start
 and end sizes that must remain positive.
 
-\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
-	I'm not making an attempt yet to revise that later content--I think other authors should
-	weigh in.}
+%\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
+%	I'm not making an attempt yet to revise that later content--I think other authors should
+%	weigh in.}
 
-\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
+%\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
 
 Within a population, we implicitly assume that allele frequency dynamics can
-be described by the Wright-Fisher model.  Briefly, generations are non-overlapping (all parents
-reproduce and die simultaneously) and the expected frequency (at birth) of the $i^{th}$ allele currently at frequency $p_i$
-one generation in the next generation is $p_iw_i/\bar{w}$, where $w_i$ and $\bar{w}$ are the marginal and
-mean fitnesses, respectively, properly weighted according to ancestry proportions. In this framework,
-a forward-time simulation of finite populations is equivalent to multinomial sampling of allele frequencies each
-generation (\citet[][pp 29-31]{burger2000-ul}, \citet[][pp 179-181]{crowkimura1970})
-and a backwards-time (coalescent) simulation follows the approximations described in \cite[][chapter 3]{wakeley2008-hd}.
-Further, this model assumes "soft" selection \citep{christiansen1975hard}, meaning that the dynamics of population sizes
+be described by the Wright-Fisher model.
+Briefly, generations are non-overlapping (all parents
+reproduce and die simultaneously) and the expected frequency (at birth)
+of the $i^{th}$ allele currently at frequency $p_i$
+one generation in the next generation is $p_iw_i/\bar{w}$,
```

Comment:
It's hard to parse this part of the sentence. Maybe the wording could be revised?

Reply:
I reworked this a bit, and let me know if this reads more clearly.

Reply:
Much clearer, thanks!